<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemLogOptionsWidget</class>
 <widget class="QWidget" name="SystemLogOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>311</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>System log</string>
  </property>
        <layout class="QVBoxLayout" name="verticalLayout_33">
         <item>
          <widget class="QGroupBox" name="boxSystemLog">
           <property name="title">
            <string> System log </string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_31">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_30">
              <item>
               <widget class="QLabel" name="lblSystemLogFileName">
                <property name="text">
                 <string>File name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="editSystemLogFileName"/>
              </item>
              <item>
               <widget class="QToolButton" name="btnSystemLogFileName">
                <property name="maximumSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="../qHostMonitor.qrc">
                  <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="chkSystemLogSuccessful">
              <property name="text">
               <string>Record info about successful actions</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkSystemLogFailed">
              <property name="text">
               <string>Record info about failed actions</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="boxUserOperationsLog">
           <property name="title">
            <string> User operations log  </string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_32">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_31">
              <item>
               <widget class="QCheckBox" name="checkBox_5">
                <property name="text">
                 <string>Log enabled, file name:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_2"/>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton_6">
                <property name="maximumSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string>Log</string>
                </property>
                <property name="icon">
                 <iconset resource="../qHostMonitor.qrc">
                  <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Results</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Operators</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Include list of tests when</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QListWidget" name="listWidget">
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <item>
                 <property name="text">
                  <string>Successful</string>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Rejected</string>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Filed</string>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QListWidget" name="listWidget_2">
                <item>
                 <property name="text">
                  <string>HosMonitor</string>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Local</string>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Remote</string>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Script</string>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Imports</string>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QListWidget" name="listWidget_3">
                <item>
                 <property name="text">
                  <string>Test imported from file</string>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Folder refreshed</string>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Folder resetted</string>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Folder enabled</string>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Folder desabled</string>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Folder removed</string>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Folder copied</string>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>

 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
